projects/storefrontlib/src/cms-components/checkout/components/payment-type/payment-type.component.html

```diff
@@ -0,0 +1,63 @@
+<div class="row">
+  <div class="col-md-12 col-lg-9">
+    <h3 class="cx-checkout-title d-none d-lg-block d-xl-block">
+      {{ 'checkoutProgress.paymentType' | cxTranslate }}
+    </h3>
+
+    <ng-container *ngIf="(paymentTypes$ | async)?.length; else loading">
+      <div class="form-check" *ngFor="let type of paymentTypes$ | async">
+        <input
+          class="form-check-input"
+          role="radio"
+          type="radio"
+          aria-checked="true"
+          (change)="changeType(type.code)"
+          [value]="type.code"
+          [checked]="type.code == typeSelected"
+        />
+        <label
+          class="cx-delivery-label form-check-label form-radio-label"
+          for="deliveryMode-{{ type.code }}"
+        >
+          <div class="cx-delivery-mode">{{ type.displayName }}</div>
+        </label>
+      </div>
+    </ng-container>
+
+    <label>
+      <span class="label-content">{{
+        'checkoutType.poNumber' | cxTranslate
+      }}</span>
+      <input #poNumber class="form-control" type="text" />
+    </label>
+
+    <label aria-required="true" *ngIf="typeSelected === 'ACCOUNT'">
```

Comment:
Is there a way to use some enum or constant instead of `'ACCOUNT'`?

Reply:
done